src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
 body {
   margin: 0;
+  background-color: gray;
+
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+a.home {
+  height: 40px;
+  display: block;
+}
+
+ul.nav {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.link {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  line-height: 60px;
+  position: relative;
```

Comment:
This property is only needed in `is-active`

Reply:
I'm sorry, what do you mean here? I can't really grasp it.

Reply:
oh, I am sorry, I meant `position: relative;`